packages/stacked_localisation/stacked_localisation_generator/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ environment:
 dependencies:
   build: ^1.3.0
   source_gen: ^0.9.6
-  analyzer: ^0.39.17
+  analyzer:
```

Comment:
I think it's a bad practice to depend on a package without a version, If you tried to publish the package with `flutter pub publish --dry-run` it will show a warning message about this
https://medium.com/cashify-engineering/flutter-dependencies-resolution-2faecdbfce03